pyproject.toml

```diff
@@ -24,6 +24,9 @@ dependencies = [
     "pydantic==2.7.3",
     "filelock==3.14.0",
     "psutil==5.9.8",
+    "tb-mqtt-client>=1.9.9",
+    "mmh3>=4.1.0",
+    "pymmh3>=0.0.5",
```

Comment:
What is `mmh3` and `pymmh3` needed for?

Reply:
I had to add them manually. Seems to be an issue with mb-mqtt-client: https://github.com/thingsboard/thingsboard-python-client-sdk/issues/60